tests/v2/unit/adapters/datumaro/adapter/test_anomaly_adapter.py

```diff
@@ -3,14 +3,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 from pathlib import Path
-from typing import TYPE_CHECKING
+from typing import Dict, TYPE_CHECKING
```

Comment:
I think we should have the following at the top and use `dict`. Otherwise, the tests would fail
```python
from __future__ import annotations
```

Reply:
I just removed `Dict` at c865c3bbf6de80842f54f20521b95508adbce851.